package.json

```
@@ -0,0 +1,54 @@
+{
+  "name": "ecommerce-application",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "preview": "vite preview",
+    "lint": "eslint ./src && npx stylelint ./src/**/*.scss",
+    "lint:fix": "eslint --fix ./src && npx stylelint ./src/**/*.scss --fix",
+    "format": "prettier ./src --write",
+    "ci:format": "prettier ./src --check",
```

```suggestion
    "format:ci": "prettier ./src --check",
```
is better as you may have a lot of `format:` and other scripts and they will be "grouped" under each category